src/test/java/core/basesyntax/model/FruitTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTest {
```

Comment:
You can leave this , but I don't see the point of testing simple getters, setters and constructors